crates/swc_ecma_preset_env/data/corejs3/builtin.json

```diff
@@ -3259,6 +3261,8 @@
       "pure": "instance/find",
       "global": [
         "es.array.find",
+        "es.iterator.find",
```

Comment:
`map`/`filter`/`flatMap`/`reduce`/`forEach`/`some`/`every` are also affected; they should be updated as well.